app/models/user.rb

```diff
@@ -241,9 +242,14 @@ def can_super_admin?
   #
   # Returns Boolean
   def can_org_admin?
-    self.can_grant_permissions? || self.can_modify_guidance? ||
-      self.can_modify_templates? || self.can_modify_org_details? ||
-      self.can_review_plans?
+    return true if can_super_admin?
+
+    # Automatically false if the user has no Org or the Org is not managed
+    return false unless org.present? && org.managed?
```

Comment:
We validate the presence of the user's org, so not sure if the `org.present?` is needed

Reply:
This occurred for me with an invite record. I don't recall the exact context in which this method was called.